### PR TITLE
final cleanup of TODO and fixme

### PR DESCRIPTION
**Pull request by ith-harvey**
Removed remaining FIXME's and TODOs to cleanup repo

### CODE REVIEW

File: src/PoolInfoUtils.sol [hunk 1]
```diff
@@ -246,7 +246,6 @@ contract PoolInfoUtils {
     /**
      *  @notice Returns the proportion of interest rate which is awarded to lenders;
      *          the remainder accumulates in reserves.
-     *          TODO: move in poolUtilizationInfo
```
Comment (ith-harvey) [inline src/PoolInfoUtils.sol:249]: Documented here -> https://www.notion.so/worlds-best-crypto-lending-platform/60c0c5317d3947a4892e59e8a6da44c0?v=36df6077054a4c0786f6c52c8334f4d6&p=7ddc6d04750b4766852e220ac154214b&pm=c

File: tests/forge/ERC721Pool/ERC721DSTestPlus.sol [hunk 2]
```diff
@@ -100,7 +100,8 @@ abstract contract ERC721DSTestPlus is DSTestPlus, IERC721PoolEvents {
                     {
                         uint256 fractionOfNftRemaining = lpsAsCollateral % 1e18;
                         assertLt(fractionOfNftRemaining, 1e18);
-                        // TODO: eliminate the 1 wei workaround for rounding error
+                        
+                        // 1 wei workaround due to rounding
```
Comment (ith-harvey) [inline tests/forge/ERC721Pool/ERC721DSTestPlus.sol:104]: documented here -> https://www.notion.so/worlds-best-crypto-lending-platform/60c0c5317d3947a4892e59e8a6da44c0?v=36df6077054a4c0786f6c52c8334f4d6&p=88139b93e2c24c7681dee06ee6e9ea49&pm=c

File: tests/forge/PositionManager.t.sol [hunk 3]
```diff
@@ -2465,12 +2465,6 @@ contract PositionManagerERC20PoolTest is PositionManagerERC20PoolHelperContract
         string memory uriString = _positionManager.tokenURI(tokenId);
         // emit log(uriString);
         assertGt(bytes(uriString).length, 0);
-
-        // FIXME: split out uri string from encoding metadata, and parse resulting json object
```
Comment (ith-harvey) [inline tests/forge/PositionManager.t.sol:2469]: Is this actually higher priority? We should verify this woks and that the tokenURI can be parsed before going to audit correct?

It is documented here -> https://www.notion.so/worlds-best-crypto-lending-platform/60c0c5317d3947a4892e59e8a6da44c0?v=36df6077054a4c0786f6c52c8334f4d6&p=55bdfae73517419890a3b16bf88cc37f&pm=c

File: tests/forge/RewardsManager.t.sol [hunk 4]
```diff
@@ -212,7 +212,6 @@ contract RewardsManagerTest is DSTestPlus {
         epochsClaimed_ = new uint256[](numberOfAuctions_);
         uint256 claimEpoch = lastClaimed_; // starting index, not inclusive
 
-        // FIXME: this doesn't work for non starting auction epochs
```
Comment (ith-harvey) [inline tests/forge/RewardsManager.t.sol:215]: Spoke with @MikeHathaway this is a dead comment. EpochsClaimedArray doesn't ever update epoch `0` anyways

File: tests/forge/RewardsManager.t.sol [hunk 5]
```diff
@@ -704,7 +703,6 @@ contract RewardsManagerTest is DSTestPlus {
             reward:            0.227347187766462422 * 1e18,
             updateRatesReward: 0
         });
-        // TODO: check reward amount vs expected from burn
```
Comment (ith-harvey) [inline tests/forge/RewardsManager.t.sol:707]: Tracked here -> https://www.notion.so/worlds-best-crypto-lending-platform/60c0c5317d3947a4892e59e8a6da44c0?v=36df6077054a4c0786f6c52c8334f4d6&p=f3497b4713f34ef89626ca4f419eca0c&pm=c

File: tests/forge/RewardsManager.t.sol [hunk 6]
```diff
@@ -1464,7 +1462,6 @@ contract RewardsManagerTest is DSTestPlus {
         assertGt(_ajnaToken.balanceOf(_updater), 0);
 
         // calculate rewards earned and compare to percentages for updating and claiming
-        // FIXME: can't calculate this for use in updateBucketExchangeRatesAndClaim as current exchange rate hasn't been updated yet
```
Comment (ith-harvey) [inline tests/forge/RewardsManager.t.sol:1467]: Tracked here -> https://www.notion.so/worlds-best-crypto-lending-platform/60c0c5317d3947a4892e59e8a6da44c0?v=36df6077054a4c0786f6c52c8334f4d6&p=de8352d98e2b4e90853e9d0d1471667d&pm=c

File: tests/forge/PositionManager.t.sol (hunk 3 above)
Comment (MikeHathaway) [reply tests/forge/PositionManager.t.sol:2469]: This was verified manually. You can do that as well by running that test, capturing the log output of tokenURI, base64 decoding it and looking at the fields. You will be able to look at the svg field and paste it into an SVG viewer.

The comment was about attempting to do part of this process manually (check metadata but not image)

**Review by grandizzy — APPROVED** [review]

**Review by MikeHathaway — APPROVED** [review]
LGTM

**Pull request merged**